src/bluesky/callbacks/broker.py

```diff
@@ -1,117 +1,13 @@
-import os
 import time as ttime
 
-import numpy as np
-
 from ..utils import ensure_uid
-from .core import CallbackBase
-
-
-class BrokerCallbackBase(CallbackBase):
```

Comment:
Why did this get moved to the other file?

Reply:
made sense to me at a time, I'm not opinionated about this now, as I mentioned above ('Discussion needed to validate file structure choices.')